src/Disks/ObjectStorages/AzureBlobStorage/AzureBlobStorageCommon.cpp

```diff
@@ -108,7 +109,7 @@ ListBlobsPagedResponse ContainerClientWrapper::ListBlobs(const ListBlobsOptions
     new_options.Prefix = blob_prefix / options.Prefix.ValueOr("");
 
     auto response = client.ListBlobs(new_options);
-    auto blob_prefix_str = blob_prefix.empty() ? "" : blob_prefix.string() + "/";
+    auto blob_prefix_str = fs::path(blob_prefix / "").string();
```

Comment:
```suggestion
    String blob_prefix_str = blob_prefix / "";
```

Reply:
I've noticed that `blob_prefix` is already a `fs::path`.

Reply:
so we don't even need ot call `fs::path` at all ?